### DISCUSSION TITLE
Native withdraw money option

### DISCUSSION THREAD

**Discussion by pipe1800** [inspireui/support#5289] [Working on Features]
It would really nice if users could make a withdraw money request from the app native settings

**Comment by minhcasi** [comment]


**Reply by bakpaorabbit** [reply]
it seems that what is meant is the Terwallet withdrawal. it need additional plugin withdrawal.

**Comment by krrisharora** [comment]
This feature is not yet implemented. We want Terawallet money withdrawal feature in app
